io/include/traccc/io/csv/measurement.hpp

```diff
@@ -38,4 +42,64 @@ struct measurement {
                    var_theta, var_time);
 };
 
+inline traccc::measurement make_measurement_edm(
+    const traccc::io::csv::measurement& csv_meas,
+    const silicon_detector_description::host* dd) {
```

Comment:
Ditto.

Reply:
Made a header and source for make_measurement_edm